### ISSUE TITLE
Request for help: how do I sum all cells selected in visual mode (sorry)?

### ISSUE THREAD

**Issue by mkastner** [andmarti1424/sc-im#651]
I apologize up front for this admittedly stupid question. I know, I shouldn't misuse the issues board for help requests.

But I am unable to find any documentation on how to create a sum from a selected range of cells in visual mode. I sometimes tend to read sloppily, so most likely, this is very much my own mistake, that I haven't found the respective section in the help documentation.

Nevertheless: would anyone here be so kind an help point me to where I can find some information on how I can sum a range of cells selected in visual mode?

**Comment by 1dancook** [comment]
tldr: set a mark for the range, then insert that mark in your expression

---

One way to accomplish this is to mark the selected range using `m` + a lower case letter. In visual mode, you can select a range and for example type `ma` to set `a` to hold the reference to that range.

Then, move to the cell in which you want to sum the range. Type `=@sum(<C-r>a)`

<C-r> = control + r

In insert mode <C-r> allows you to insert the range or reference that is stored in a mark.

These things are written about in the :help document but it takes a little practice to piece the different commands and tools together. 

I'm not aware of another way to do this other than manually typing the range.

**Comment by 1dancook** [comment]
By the way, there is a discussion section that you can use 

https://github.com/andmarti1424/sc-im/discussions

If you need help check there or ask 👍😊

**Comment by mkastner** [comment]
Thanks a lot for your help. I wasn't aware of the forum, but it's definitely the place I should have placed my questions.

So, again thank you very much an also – again – I apologize for having misused this issues section. BTW sc-im is awesome. It frees me of MS-Excel and I can use my favourite Vim keys. Thanks to everyone involved in this project.

**Comment by andmarti1424** [comment]
Thanks @1dancook
